djangocms_versioning/models.py

```diff
@@ -95,6 +95,19 @@ class Meta:
     def __str__(self):
         return "Version #{}".format(self.pk)
 
+    def delete(self, using=None, keep_parents=False):
```

Comment:
Can you add a comment or docstring to explain why we are deleting here. We are seriously lacking comments in the code.